truss/truss_handle.py

```diff
@@ -76,7 +76,8 @@ def docker_run(
                 'type=bind',
                 f'src={str(secrets_mount_dir_path)}',
                 'target=/secrets',
-            ]]
+            ]],
+            runtime='nvidia' if self._spec.config.resources.use_gpu else None,
```

Comment:
I read that this was the old way. I think there is a new flag: --gpus all 

https://docs.docker.com/engine/reference/commandline/run/
https://stackoverflow.com/questions/25185405/using-gpu-from-a-docker-container

that is used for Docker 19.03. Otherwise looks good

Reply:
nice, great suggestion. Let me update.